CONTRIBUTING.txt

```diff
@@ -222,13 +221,13 @@ Guidelines
 * All code should have tests (see `test coverage`_ below for more details).
 * All code should be documented, to the same
   `standard <https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard>`_ as NumPy and SciPy.
-* For new functionality, always add an example to the gallery.
+* For new functionality, always add an example to the gallery (see
+  :ref:`sphinx_gallery` below for more details).
```

Comment:
Isn't it
```suggestion
  :ref:`_sphinx_gallery` below for more details).
```
?

Reply:
the updated link works, resolving this convo

Reply:
That's also what @jni was suggesting :thinking: but... see https://github.com/scikit-image/scikit-image/pull/4857#discussion_r461400636 and https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#use-a-custom-anchor